client/Helpers/UI.elm

```diff
@@ -113,11 +112,15 @@ grid columns rows =
 
 {-| Create 'onselect' event listener attribute for select HTML node
 -}
-onSelect : (String -> msg) -> Html.Attribute msg
-onSelect msg =
+onChange : (String -> msg) -> Html.Attribute msg
```

Comment:
This now differs from the documentation line above

Reply:
`onSelect` is redundant now. removed